packages/gatsby/cache-dir/production-app.js

```diff
@@ -80,14 +80,30 @@ apiRunnerAsync(`onClientEntry`).then(() => {
     }
   }
 
+  if (
+    window.page.path &&
+    window.page.path !== window.location.pathname &&
+    window.page.path !== `/offline-plugin-app-shell-fallback/`
+  ) {
+    const url = new URL(window.location)
+    url.pathname = window.page.path
+    navigate(url, { replace: true })
```

Comment:
`navigate` only takes a string https://github.com/reach/router/blob/master/website/src/markdown/api/navigate.md#to

Reply:
Thanks for pointing this out, just fixed it!